.vscode/settings.json

```diff
@@ -16,5 +16,6 @@
     "./.moon/cache/schemas/workspace.json": [
       ".moon/workspace.yml"
     ]
-  }
+  },
+  "typescript.tsdk": "node_modules/typescript/lib"
```

Comment:
again, ergonomics.

Sometimes vscode will have a different enough version of typescript to the project, ending up with non obvious errors.